components/filter-components/MapFilters.vue

```diff
@@ -183,13 +183,16 @@
                       {{ propertyItem.label }}
                     </p>
                     <div
-                      class="inline-flex items-center justify-center align-middle rounded-full border-r-2 border-l-2 border-gray-400 w-5 h-5 cursor-pointer"
+                      class="inline-flex items-center justify-center align-middle w-5 h-5 ml-1 mb-1 cursor-pointer"
                     >
                       <img
-                        src="/images/infoIcon.svg"
-                        alt="Filter Information Popup"
+                        src="/images/questionIcon.svg"
+                        aria-label="Filter Description Tooltip"
```

Comment:
I think we should remove the word tooltip but otherwise good. Thanks Ruby

Reply:
done

Reply:
thanks Ruby